pkg/postgres/query/builder_test.go

```diff
@@ -404,19 +409,41 @@ func TestQueryBuilder(t *testing.T) {
 						{
 							Name:     "other_filter_field",
 							Operator: filter.CompareOperatorBeginsWith,
-							Value:    []any{"B%SI"},
+							Value:    []any{"B%SI", "T%logy"},
 						},
 						{
 							Name:     "status",
-							Operator: filter.CompareOperatorBeginsWith,
+							Operator: filter.CompareOperatorContains,
 							Value:    []any{"S%TI"},
 						},
 					},
 					Operator: filter.LogicOperatorOr,
 				},
 			},
-			wantQuery: `WHERE ("other_filter_field_col_name" ILIKE ? || '%') OR ("status_col_name" ILIKE ? || '%')`,
-			wantArgs:  []any{`B\%SI`, `S\%TI`},
+			wantQuery: `WHERE ("other_filter_field_col_name" ILIKE ? || '%') OR ("other_filter_field_col_name" ILIKE ? || '%') OR ("status_col_name" ILIKE '%' || ? || '%')`,
+			wantArgs:  []any{`B\%SI`, `T\%logy`, `S\%TI`},
+		},
+		{
+			name: "build query with filter compare operators 'is greater than' && 'is less than'",
```

Comment:
title claims there is `is less than`, but it is not used in the `Filter` below

Reply:
Changed now

Reply:
filters used in this testcase are only `filter.CompareOperatorIsGreaterThanOrEqualTo`, so it does not fit yet

Reply:
Checkout [e80df38](https://github.com/greenbone/opensight-golang-libraries/pull/77/commits/e80df38bc96c3af01baaa4b00c7c6706aa47293d)